roles/workstation/tasks/file-config.yml

```diff
@@ -86,3 +86,11 @@
 
 - name: Flush handlers
   meta: flush_handlers
+
+- name: enable sysrq key
+  ansible.builtin.copy:
+    dest: "/etc/sysctl.d/99-reisub.conf"
+    content: |
+      # enable sysrq key to hard reboot: https://forum.endeavouros.com/t/tip-enable-magic-sysrq-key-reisub/7576
+      kernel.sysrq = 1
+  notify: reload sysctl
```

Comment:
This requires a sysctl handler in handlers to work.

Reply:
Sorry i renamed the handler since it didn't actually reload the systcl, it just made sysctl read a config file in order for the feature to work.  I just pushed the renamed handler that matches the one in handlers/main.yml